docs/src/PolyhedralGeometry/cones.md

```diff
@@ -43,19 +43,14 @@ Objects of type `Cone` can be saved to a file and loaded from a file in the
 following way:
 ```@repl oscar
 C = positive_hull([1 0; 0 1])
-save_cone(C, "C.cone")
-CC = load_cone("C.cone")
+save(C, "C.cone")
+CC = load("C.cone")
```

Comment:
Should this perhaps be
```suggestion
CC = load(Cone, "C.cone")
```
?

Reply:
So the question is, whether we can have both, i.e. `load(filename)` and `load(Type, filename)`. The latter for places where type inference matters for performance reasons, the former for convenience, i.e. inexperienced users, we don't have to explain to people how to figure out what is in the file or how to tell their colleagues. I now added both, and I think it doesn't hurt to have `load(filename)` for now. If it becomes too unmanageable we can always eliminate it. Is there some strong reason against that I don't see?